cfgov/unprocessed/css/video-player.less

```diff
@@ -121,6 +121,16 @@
     margin: auto;
 }
 
+.video-player_iframe-container {
+    height: 100%;
+    margin: 0;
+}
+
+.video-player_video-actions-container {
+    right: 0;
+    left: auto;
+}
+
 .respond-to-min( @bp-sm-min, {
     .video-player_video-actions-container {
```

Comment:
This might just be some sort of cached style I'm not clearing right, but when I test out FCM videos, the close button looks wonky at all screen sizes:

![wonky-close](https://user-images.githubusercontent.com/1862695/60129151-60e94400-9762-11e9-8cc4-3b9190ba9156.png)

Looks like maybe the `left: auto;` is getting overridden by the `left: @video-width__px;` in this media query? Alternatively, it's 100% possible I didn't build something right and this isn't actually a problem.

Reply:
Nooooope you're right. I'll spend some more time with this.